game/templates/game/board.html

```
@@ -511,6 +511,12 @@
                    </div>
                </div>
                 
+                <!-- Quick Move Input (SAN) -->
+                <div class="san-input-container" style="margin-top: 15px; display: flex; gap: 8px;">
+                    <input type="text" id="sanMoveInput" placeholder="Move (e.g. e4, Nf3, O-O)" aria-label="Enter move notation" style="flex: 1; padding: 10px; border-radius: 5px; border: 1px solid #444; background: #1a1a2e; color: #fff; font-size: 0.9rem;" autocomplete="off">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)